pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -264,11 +264,11 @@ func addAddons(files *[]assets.CopyableFile, data interface{}) error {
 }
 
 // waitForPods waits until the important Kubernetes pods are in running state
-func waitForPods(k8s config.KubernetesConfig, quiet bool) error {
+func waitForPods(k8s config.KubernetesConfig, quiet bool, componentsOnly bool) error {
```

Comment:
Having two bool arguments to a function is a bad code smell. Do you mind splitting this out into two functions? Perhaps something like:

waitForComponentPods and waitFork8sPods?

Reply:
OK